lib/generate-svg.js

```diff
@@ -128,17 +134,23 @@ module.exports = (sources = [], options = {}, compilation) => {
                 return source.content;
             })();
 
-            SVGOptimizer.optimize(sprite, {
-                path: source.path
-            }).then((output) => resolve({
-                prefix: prefix,
-                file: source.path,
-                id: id,
-                sprite: output.data
-            })).catch((error) => {
-                addWarning(new SpriteParsingWarning(`Sprite '${id}' could not be optimized.\n${error}`), compilation);
-                resolve();
-            });
+            // try {
+                const output = svgo.optimize(sprite, {
+                    path: source.path,
+                    ...svgoConfig
+                })
+                resolve({
+                    prefix: prefix,
+                    file: source.path,
+                    id: id,
+                    sprite: output.data
+                })
+            // } catch (error) {
+            //     console.error(error)
+            //     console.log("source", sprite)
+            //     addWarning(new SpriteParsingWarning(`Sprite '${id}' could not be optimized.\n${error}`), compilation);
+            //     resolve();
+            // }
```

Comment:
@cascornelissen I had trouble debugging an issue with svgo-merge and the error was being swallowed here. This should probably be uncommented.